File: Java programs/Problems on Pattern Printing/String Pattern/Pattern5.java
/* Write a java program which accept String from user and display below pattern.

Input : Hello
Output :  H  #   #   #   #
          H  e   #   #   #
		  H  e   l   #   #
		  H  e   l   l   #  
		  H  e   l   l   0      
		  H  e   l   l   #
          H  e   l   #   #
		  H  e   l   #   #
		  H  e   #   #   #     
		  H  #   #   #   #         

*/
		  
import java.util.*;

class Pattern
{
    public void Pattern(String str)
    {
	    char Arr[]=str.toCharArray();
	    int i=0,j=0;
       	    
        for(i=0;i<Arr.length-1;i++)        
	    {
			for(j=0;j<Arr.length;j++)
		    {
			   if(i>=j)
			   {
			      System.out.print(Arr[j]+"\t");
			   }
			   else
			   {
				   System.out.print("#"+"\t");
			   }   
		    }	
		    System.out.println();
        }
	   
	    for(i=Arr.length-1;i>=0;i--)        
	    {
			for(j=0;j<Arr.length;j++)
		    {
			    if(i>=j)
			    {
			      System.out.print(Arr[j]+"\t");
			    }
				else
				{
					System.out.print("#"+"\t");
				}	
		    }	
		    System.out.println();
		}	
    }
}

class Lba395
{
	public static void main(String arg[])
	{
		Scanner sobj=new Scanner(System.in);
		
		System.out.println("Enter string");
		String str=sobj.nextLine();
		
		Pattern pobj=new Pattern();
		pobj.Pattern(str);
	}	
}

File: C  programs/Problems on Pattern Printing/Pattern5.c
/* Write a program accept number from user and Display below pattern.

Input: 6
Output: * # * # * # 

*/

#include<stdio.h>

void DisplayPattern(int iNo)
{
   int iCnt=0;

   if(iNo<0)
   {
   	 iNo=-iNo;
   }

   for(iCnt=1;iCnt<=iNo;iCnt++)
   {
	  if((iCnt%2)==0)
	  {
	     printf("#\t");
	  }
      else
      {
        printf("*\t");
      }		
   }
}

int main()
{
   int iValue=0;

   printf("Enter the number\n");
   scanf("%d",&iValue);

   DisplayPattern(iValue);

   return 0;
}   